src/Life/Main/Init.hs

```diff
@@ -1,28 +1,49 @@
+{-# LANGUAGE TemplateHaskell #-}
+
 -- | Contains function to create repository.
 
 module Life.Main.Init
        ( lifeInit
        ) where
 
-import Path.IO (copyFile, doesFileExist)
+import Path (mkRelFile)
+import Path.IO (copyFile, doesDirExist, doesFileExist)
 
-import Life.Configuration (lifePath, singleFileConfig, writeGlobalLife)
+import Life.Configuration (LifeConfiguration (..), lifePath, renderLifeConfiguration,
+                           singleFileConfig, writeGlobalLife)
 import Life.Github (Owner, Repo (Repo), createRepository, doesRepoExist, insideRepo, repoName)
-import Life.Message (chooseYesNo, errorMessage, infoMessage, warningMessage)
+import Life.Message (chooseYesNo, errorMessage, infoMessage, skipMessage, successMessage,
+                     warningMessage)
 import Life.Shell (createDirInHome, relativeToHome)
 
+import qualified Data.Set as Set
+
+predefinedLifeConfig :: LifeConfiguration
+predefinedLifeConfig = mempty
+    { lifeConfigurationFiles = Set.fromList
+          [ $(mkRelFile ".bash_profile")
+          , $(mkRelFile ".profile")
+          , $(mkRelFile ".vimrc")
+          , $(mkRelFile ".emacs")
+          , $(mkRelFile ".spacemacs")
+          , $(mkRelFile ".gitconfig")
+          , $(mkRelFile ".ghc/ghci.conf")
+          , $(mkRelFile ".stylish-haskell.yaml")
```

Comment:
Please, feel free to give any suggestions what else can be added to this default list.